src/consts.ts

```diff
@@ -13,37 +13,9 @@ export const FIELDS = {
   },
 };
 
+export type Fields = typeof FIELDS;
+
 export enum NETWORK {
   ETH_SEPOLIA = "eth-sepolia",
```

Comment:
need `eth-sepolia-fork` so that it saves it into the db appropriately

need to change the language from `*-local` to `*-fork` everywhere else too